state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -1157,6 +1157,14 @@ pub struct PoolInfo {
 	/// The fee taken, when range orders are used, from swap inputs that contributes to liquidity
 	/// provider earnings
 	pub range_order_fee_hundredth_pips: u32,
+	/// The total fees earned in this pool by range orders.
+	pub range_order_total_fees_earned: SideMap<Amount>,
```

Comment:
Not sure about the return value here. Maybe something scalar would make more sense? Same for all the over new added filelds.

Reply:
So we can use SideMap as you have here, and in the other PR of yours we will change it to PairMap (or whatever we call it).